usaspending_api/common/helpers/s3_helpers.py

```diff
@@ -37,3 +39,19 @@ def access_s3_object(bucket_name: str, obj: "boto3.resources.factory.s3.ObjectSu
     bucket.download_fileobj(obj.key, data)
     data.seek(0)  # Like rewinding a VCR cassette
     return data
+
+
+def upload_download_file_to_s3(file_path):
+    bucket = settings.BULK_DOWNLOAD_S3_BUCKET_NAME
+    region = settings.USASPENDING_AWS_REGION
+    multipart_upload(bucket, region, str(file_path), file_path.name)
+
+
+def multipart_upload(bucketname, regionname, source_path, keyname):
+    s3client = boto3.client("s3", region_name=regionname)
+    source_size = Path(source_path).stat().st_size
+    # Sets the chunksize at minimum ~5MB to sqrt(5MB) * sqrt(source size)
```

Comment:
Is there an important reason for this 5MB sized chunk? Is this dependent on any AWS settings? If so, probably a good idea to at least note that here.

Reply:
Pre-existing code. I'm not going to spend time today tracking down the justification.